### PR TITLE
[MDE-2021] Updating logo of the event and adding sponsors (updating logo for Ceiba)

### PR DESCRIPTION
**Pull request by mpinedag12**
[MDE-2021] Updating logo of the event and adding sponsors (updating logo for Ceiba)

### CODE REVIEW

File: data/sponsors/ngix2021.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+name: "ngix2021"
```
Comment (phrawzty) [inline data/sponsors/ngix2021.yml:1]: Again, the name of the company is "Nginx", but moreover, the process for changing a logo for a sponsor is documented here: https://github.com/devopsdays/devopsdays-web/blob/main/utilities/README.md#updating-a-sponsor-image-or-data

Please use that method to change the sponsor logo. If you need any help, ping us in #websites on the Slack, or comment here. 👍

File: data/events/2021-medellin.yml [hunk 2]
```diff
@@ -171,6 +173,14 @@ sponsors:
     level: gold
   - id: castor
     level: gold
+  - id: unijaveriana
+    level: gold
+  - id: accenture
+    level: gold
+  - id: masglobal
+    level: gold
+  - id: ngix2021
```
Comment (phrawzty) [inline data/events/2021-medellin.yml:182]: The name of the company in Nginx, not `ngix`.

**Review by phrawzty — CHANGES_REQUESTED** [review]

**Review by phrawzty — APPROVED** [review]

File: data/events/2021-medellin.yml [hunk 3]
```diff
@@ -171,6 +173,14 @@ sponsors:
     level: gold
   - id: castor
     level: gold
+  - id: unijaveriana
+    level: gold
+  - id: accenture
+    level: gold
+  - id: masglobal
+    level: gold
+  - id: nginx
```
Comment (phrawzty) [inline data/events/2021-medellin.yml:182]: One last thing: please remove `static/img/sponsors/ngix2021.png ` from this PR as it's no longer used. We'll be good to merge after that!

**Review by phrawzty — CHANGES_REQUESTED** [review]

File: data/events/2021-medellin.yml (hunk 3 above)
Comment (bridgetkromhout) [reply data/events/2021-medellin.yml:182]: I'll remove it

**Pull request merged**